ui/pages/send/send.utils.js

```diff
@@ -25,7 +28,12 @@ function isBalanceSufficient({
   gasTotal = '0x0',
   primaryCurrency,
 }) {
-  let totalAmount = new Numeric(amount, 16).add(new Numeric(gasTotal, 16));
+  let totalAmount;
+  if (amount === '0x0') {
+    totalAmount = new Numeric(amount, 16);
+  } else {
+    totalAmount = new Numeric(amount, 16).add(new Numeric(gasTotal, 16));
+  }
```

Comment:
What was this change made for?

Reply:
Well if we add the gas fee to the amount when amount is 0, then total amount will always be greater then 0, and even though user has 0 balance and didnt type any amount, the insufficient funds error would be shown and that should not be the case, because there is warning message on the bottom that he does not have enough tokens for transactions. I think insufficient funds error should only be shown when user starts typing amount, not when amount is 0.